packages/babel-preset-env/src/debug.js

```diff
@@ -15,9 +15,23 @@ export const logMessage = (message, context) => {
 export const logPlugin = (plugin, targets, list, context) => {
   const envList = list[plugin] || {};
   const filteredList = Object.keys(targets).reduce((a, b) => {
```

Comment:
Not related to this change, but `a` and `b` should really be renamed.

Reply:
Cleaned it up!